examples/pruner/job.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: image-prune
+spec:
+  schedule: 0/15 * * * ?
```

Comment:
Can we have less frequent one? I'd say once an hour to start with. That will at least create 4 times less jobs than every quarter of an hour.